### PR TITLE
Create ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by jarifibrahim**
Related to https://github.com/dgraph-io/badger/pull/831

This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/badger/834)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+<pre>
+$ go version
+
+</pre>
+
+### What version of Badger are you using?
+
+
+### Does this issue reproduce with the latest release?
```
Comment (ashish-goswami) [inline .github/ISSUE_TEMPLATE]: We can also ask to reproduce it on latest master.

Comment (jarifibrahim) [reply .github/ISSUE_TEMPLATE]: Done

**Comment by campoy** [comment]
Why did the TeamCity test fail with this change?
Does this mean we have flaky tests?

**Comment by jarifibrahim** [comment]
@campoy The build failed because of https://github.com/dgraph-io/badger/issues/793

**Pull request merged**